File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/ProductOptions/ProductOptionBasic.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ohtap.ECommerce.Domain.Models.Objects.ProductOptions
{
    public class ProductOptionBasic : IProductOptionData
    {
        public virtual string ID { get; set; }
        public virtual string Code { get; set; }
        public virtual string Name { get; set; }
        public virtual int Order { get; set; }
        public virtual string Description { get; set; }
        public virtual int State { get; set; }
        public virtual string Value { get; set; }
        public virtual Options.IOptionData Option { get; set; }
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/ProductOptions/ProductOptionBasic.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Promotions;




use Ohtap\ECommerce\DomainModels\Objects\Options\IOptionData;

class ProductOptionBasic implements IProductOptionData
{

    public string $Description;
    public IOptionData $Option;
    public string $Value;
    public int $State;
    public int $OrderNo;
    public string $Name;
    public string $ID;

}